packages/jest-jasmine2/src/queueRunner.js

```diff
@@ -32,8 +32,8 @@ function queueRunner(options: Options) {
   const mapper = ({fn, timeout}) => {
     const promise = new Promise(resolve => {
       const next = once(resolve);
-      next.fail = (...args) => {
-        options.fail(...args);
+      next.fail = () => {
+        options.fail.apply(null, arguments);
```

Comment:
Arrow functions don't bind `arguments`. If this is run through babel and converted to a normal function I guess it's fine, but then why was spread angry? 🤔